models/articles.models.js

```diff
@@ -28,6 +28,8 @@ exports.selectArticles = (sort_by = "created_at", order = "desc") => {
 
   const validOrder = ["ASC", "DESC"];
 
+  const validTopic = ["paper", "cats", "mitch"];
```

Comment:
Are you sure you want to hardcode the available topics? Have you looked at the topics in the dev data?